astropy/extern/ply/lex.py

```diff
@@ -41,8 +41,6 @@
 import os
 import inspect
 
-from ...extern.six.moves import zip
```

Comment:
I think it's better to add this upstream, especially that this is purely stylistic change, they have a try/except workaround.